Competition/src/test/java/competition/subsystems/gripper/GrabDiscCommandTest.java

```diff
@@ -0,0 +1,34 @@
+package competition.subsystems.gripper;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import competition.BaseCompetitionTest;
+import competition.subsystems.gripper.commands.GrabDiscCommand;
+
+public class GrabDiscCommandTest extends BaseCompetitionTest {
+
+    GrabDiscCommand grabDisc;
+    GripperSubsystem gripperSubsystem;
+    @Override
+    public void setUp() {
+        super.setUp();
+        grabDisc = this.injector.getInstance(GrabDiscCommand.class);
+        gripperSubsystem = this.injector.getInstance(GripperSubsystem.class);
+    }
+
+    @Test
+    public void testGrabDisc() {
+        gripperSubsystem.gripperPiston.setOn(false);
+        assertFalse("grabHatchet is false", gripperSubsystem.gripperPiston.getAdjusted());
+
+        gripperSubsystem.grabHatch();
```

Comment:
This method doesn't actually use the GrabDiscCommand. You should call `GrabDiscCommand.initialize()`